WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift

```diff
@@ -236,6 +236,11 @@ import Reachability
     /// Public method to hide/show the image view.
     ///
     @objc func hideImageView(_ hide: Bool = true) {
+        if let isReachable = reachability?.isReachable(),
```

Comment:
Hey @danielebogo . I'm a little concerned about putting this here. `hideImageView` is a public method that is called to essentially force the hide the image. A cursory search shows this is typically done based on device orientation, especially for iPhone landscape where there isn't a lot of space. On the views that expect the image to be hidden, it will instead be shown for no connection, which probably won't look very nice. 

Thoughts?

Reply:
Ah! Good catch! Yes in that case I have to remove that.
I have a question about this. In `func configureView()`, line 311, can the `imageView.image` be nil by default?

Reply:
Nevermind @ScoutHarris ! I came up with another solution!

Reply:
I fixed it in [7ae9a3f](https://github.com/wordpress-mobile/WordPress-iOS/pull/11654/commits/7ae9a3f1fc893fe1a2c021b97b9f110722387b3d).
You were right and that check was out of scope.